.eslintrc.json

```diff
@@ -28,10 +28,23 @@
   // Specify the ESLint plugins tobe used
   "plugins": ["react", "@typescript-eslint", "react-hooks", "jest"],
   "rules": {
-    "react/destructuring-assignment": ["warn", "always"],
```

Comment:
@yasharth291 are the changes to `.eslintrc.json` acceptable? I'd prefer to make the linting more thorough, and I'm not sure this is the case here?

Reply:
I think this will not change anything as it is only warning